### DISCUSSION TITLE
int64

### DISCUSSION THREAD

**Discussion by tgkolda** [sandialabs/pyttb#377] [General]
I notice there are a couple pf places that np.int64's are being used rather than np.int's. I was wondering about the reasoning for this.

**Comment by ntjohnson1** [comment]
Without a reference to where you saw this there are a few different possible answers:
1. `np.int` doesn't name a constructable type https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
1.1 Numpy types are generally sized (or platform dependent) https://numpy.org/doc/stable/user/basics.types.html 
3. If you were looking at import data then I assume this is for consitency when we are loading data sourced from matlab
4. If you were looking at test_tenmat I'm not totally sure why I was explicit in the integer type there and its probably not necessary
5. If you were looking at somewhere else in the code base my grep missed something where we explicitly created an np.int64
6. If you were looking at the resulting type of an output then this was probably just because of a type promotion or type inference when numpy got an integer type (see link above for more details on types). Additionally, this may be an artifact of some of the weirdness/inconsistency you are seeing on numpy 1.26 around type print outs/type promotion.
